Test.C

```diff
@@ -235,6 +235,57 @@ int main(){
         // Selling equipment
         // decrease amount in index amount if things are sold
         // increase credit if things are sold
+        // So this is a lot of pseudo code or not finished but idk how to do it fully but it a good framework 
+        // assumes that you can sell the equipment for the same price that is bought 
+        int equipSellSelect;
+        int equipSellAmount;
+        int equipSellTransaction;
+        int equipSellAnswer;
+        
+        printf("|----------------------------- Equipment Selling ------------------------------|\n");
+        printf("Equipment that you own:\n");
+        for(int i = 0; i < NUM_EQUIPMENT; i++){
+            printf("%s %s %d %d\n", equipmentStock[i].name, equipmentStock[i].description, equipmentStock[i].cost, equipmentStock[i].messageAmount);
+            printf("Amount of %s that is in your inventory: %d", equipmentStock[i].name, player.equipmentInventory[i]);
+        }
+        printf("Your current credits are: %d\n", player.currentCredits); // current credit
+        printf("Please select the equipment you would like to sell or select %d to pass: ", NUM_EQUIPMENT + 1); // chose 11 becuase there are 10 options rn
+        scanf("\n%d", &equipSellSelect);
+
+        while (equipSellSelect < NUM_EQUIPMENT){ // while the user is selecting something to sell and not choseing to pass
+            printf("You have selected: %s", equipmentStock[0].name); // not sure if that should be equipmentStock or not
+            printf("\n How many would you like to sell? ");
+            scanf("\n%d", &equipSellAmount);
+            equipSellTransaction = equipmentStock[0][2] * equipSellAmount; // not sure if that should be equipmentStock or not
+            printf("This will sell for: %d", equipSellTransaction);
+            printf("Are you sure you want to sell? y/N");
+            scanf("\n%d", &equipSellAnswer);
+            if (equipSellAnswer == "y" || equipSellAnswer == "Y"){ 
+                if (player.equipmentInventory[0] > 0){ // checks that the user has the item in thier inventory
+                    printf("Thank You!");
+                    player.currentCredits = player.currentCredits + equipSellTransaction;
+                    printf("Your current credits are: %d\n", player.currentCredits); //current credit
+                    player.equipmentInventory[0]= player.equipmentInventory[0] - equipSellAmount; 
+                }
+                if (player.equipmentInventory == 0){ // checks that the user has the item in thier inventory
+                    printf("You do not have any of this equipment in your inventory");
+                    printf("The transcation was canceled");
+                }
+            }
+            if (equipSellAnswer == "n" || equipSellAnswer == "N"){
+                printf("The transcation was canceled");
+            }
+            // print equipment that is owned
+            for(int i = 0; i < NUM_EQUIPMENT; i++){
```

Comment:
Horizontal spacing per the style guide.
```suggestion
            for(int i=0; i<NUM_EQUIPMENT; i++){
```